roles/redis/defaults/main.yaml

```diff
@@ -0,0 +1,2 @@
+---
+redis_image: docker.io/library/redis:6
```

Comment:
We can figure this out later (maybe we file issues for things we find?). Is this image running a version of Redis licensed the way we'd want?

Reply:
No idea. We can replace it with valkey to be sure. I just needed something for a start.

Reply:
I checked, Redis changed the license in 7.4 -- so earlier versions are fine.